roles/ara_api/tasks/install/podman.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Detecting existing PyPI installation
+  ansible.builtin.stat:
+    path: "{{ ara_api_venv_path }}"
+  register: existing_pypi_install
+
+- name: Notify about existing PyPI installation
+  ansible.builtin.debug:
+    msg: |
+      You seem to have ARA-API installed via PyPI in the past, you might
+      want to clean up that installation and migrate your data
+  when: existing_pypi_install['stat']['exists']
+
+- name: Override file locations with path in container
+  ansible.builtin.set_fact:
+    ara_api_database_name: "/opt/ara/ansible.sqlite"
+    ara_api_log_dir: "/opt/ara/logs"
+    ara_api_settings: "{{ ara_api_root_dir }}/settings.yaml"
+
+- name: Ensure ARA API container
+  containers.podman.podman_container:
+    name: ara-api
+    image: "{{ ara_api_image }}:{{ ara_api_version }}"
+    pull: newer
+    state: present
+    auto_remove: true
+    generate_systemd: "{{ ara_api_systemd_config }}"
+    ports:
+      - 127.0.0.1:8000:8000
+    volume:
+      - "{{ ara_api_root_dir }}:/opt/ara:z"
```

Comment:
That extra ``:z`` at the end has caused me problems when/where selinux wasn't available (i.e, on ubuntu/debian).

We should consider adding it only if selinux is enabled (there is a fact for that): https://demo.recordsansible.org/hosts/11803.html#ansible_selinux

![image](https://github.com/user-attachments/assets/5d8af97b-8914-422e-bf73-6353e3635ddf)

Reply:
Oh, that's interesting, it hasn't caused issues on my own control node I run at home, but yes, after the CI thing is merged, I can make conditional :)

Reply:
There may have been fixes in podman (or debian/ubuntu) since I last tried this, but I mean to say that we shouldn't set the selinux bit if we know there is no selinux.